guides/notebooks/native-navigation.livemd

```diff
@@ -0,0 +1,377 @@
+
+* redirect -> sends the [`redirect` event](https://github.com/liveview-native/liveview-client-swiftui/blob/9895c3b16d84a2683dcb1f127994be6c1bdf4919/Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift#L274-L281).
+* push_navigate -> sends a [`live_redirect` event](https://github.com/liveview-native/liveview-client-swiftui/blob/9895c3b16d84a2683dcb1f127994be6c1bdf4919/Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift#L258-L265).
+* push_patch -> sends a [`live_patch` event](https://github.com/liveview-native/liveview-client-swiftui/blob/9895c3b16d84a2683dcb1f127994be6c1bdf4919/Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift#L266-L273).
+
+The `LiveViewCoordinator` [sends those navigation requests to the LiveSessionCoordinator](https://github.com/liveview-native/liveview-client-swiftui/blob/0e0fc6bbe5e95ef308e51551af0889acb09b87b3/Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift#L353) which handles them natively.
+
+The LiveView Native [LiveSessionCoordinator](https://github.com/liveview-native/liveview-client-swiftui/blob/0e0fc6bbe5e95ef308e51551af0889acb09b87b3/Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift) manages the current [navigation path](https://github.com/liveview-native/liveview-client-swiftui/blob/0e0fc6bbe5e95ef308e51551af0889acb09b87b3/Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift#L43). The navigation path contains a list of [LiveNavigationEntries](https://github.com/liveview-native/liveview-client-swiftui/blob/0e0fc6bbe5e95ef308e51551af0889acb09b87b3/Sources/LiveViewNative/Coordinators/LiveNavigationEntry.swift) which acts as the history of pages the user has viewed.
+
+<!-- livebook:{"break_markdown":true} -->
+
+### Disconnected Mount
+
+LiveViews are mounted twice to improve their performance. During the first `mount/3` callback, the template is immediately sent to the client. The socket connection is established during the second `mount/3` callback, which allows for interactivity on the page. This is applicable to both native and web applications. Currently, a loading indicator is automatically shown during the disconnected mount. However, there is an [open issue](https://github.com/liveview-native/liveview-client-swiftui/issues/1229) that may change this behavior.
+
+Be aware that:
+
+* The `redirect/2` doesn't trigger the disconnected mount as you might expect. There is an [open issue](https://github.com/liveview-native/liveview-client-swiftui/issues/1228) and this behavior may change in the future to make LiveView Native more in line with web behavior.
+* We currently don't support `live_session/3` however there is a [PR in progress](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.Router.html#live_session/3)
```

Comment:
The link to `PR in progress` is incorrect. I believe the referenced PR was merged, however. You may just need to note that cross-session navigation is only supported from the root path.